mkosi/config.py

```diff
@@ -463,6 +463,12 @@ class MkosiConfigParser:
             section="Output",
             parse=config_make_list_parser(delimiter=",", parse=make_path_parser(required=False)),
         ),
+        MkosiConfigSetting(
+            dest="pager",
+            section="Output",
+            parse=config_parse_boolean,
+            default=True,
+        ),
```

Comment:
Should this be configurable in the configuration file? I feel it's one of those options that should be CLI only.